src/tests/contract/CicCriProvider.test.ts

```diff
@@ -0,0 +1,53 @@
+import { Logger } from "@aws-lambda-powertools/logger";
+import { Verifier, VerifierOptions } from "@pact-foundation/pact";
+import { Constants } from "./utils/Constants";
+import path from "path";
+
+const logger = new Logger({
+	logLevel: "INFO",
+	serviceName: "CicCriProviderTest",
+});
+
+const pactFile = path.resolve("./tests/contract/pacts/IpvCoreBack-CicCriProviderAll.json");
+
+let opts: VerifierOptions;
+// Verify that the provider meets all consumer expectations
+describe("Pact Verification", () => {
+	beforeAll(() => {  
+		opts = {
+			// we need to know the providers name
+			provider: "CicCriProvider",
+			// we are starting the provider locally
+			providerBaseUrl: `${Constants.LOCAL_HOST}:${Constants.LOCAL_APP_PORT}`,
+			// pactBrokerUrl: process.env.PACT_BROKER_URL,
```

Comment:
Are we leaving these in and commented out until we have it avail?

Reply:
I have manually published the pactfiles and tested the verifier which has successfully published the results too there onto the pactbroker. Until we have it integrated with the pipeline, plan to leave it commented and revisit it once IPV and we start work on pipeline.